### ISSUE TITLE
Unable to create new experiment on console

### ISSUE THREAD

**Issue by jagusztinl** [kubeflow/kubeflow#6016]
/kind bug

**What steps did you take and what happened:**
[A clear and concise description of what the bug is.]

Create new experiment on console.

Error:
Experiment creation failed
{"error":"Validate experiment request failed.: Invalid input error: In single-user mode, CreateExperimentRequest shouldn't contain resource references.","code":3,"message":"Validate experiment request failed.: Invalid input error: In single-user mode, CreateExperimentRequest shouldn't contain resource references.","details":[{"@type":"type.googleapis.com/api.Error","error_message":"In single-user mode, CreateExperimentRequest shouldn't contain resource references.","error_details":"Validate experiment request failed.: Invalid input error: In single-user mode, CreateExperimentRequest shouldn't contain resource references."}]}
​

**What did you expect to happen:**
Created experiment

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Kubeflow version: (version number can be found at the bottom left corner of the Kubeflow dashboard):
- 1.3.0
- kfctl version: (use `kfctl version`):
- Kubernetes platform: (e.g. `minikube`)
- OKD 4.6
- Kubernetes version: (use `kubectl version`):
- 1,18
- OS (e.g. from `/etc/os-release`):

**Comment by DHolmanCoding** [comment]
Also experienced this issue

**Comment by hade94** [comment]
+1

**Comment by miutamihai** [comment]
+1

**Comment by OmGraja** [comment]
+1 too

**Comment by kimwnasptd** [comment]
Apologies for being completely unsynced but what exactly do you mean by `Experiment` is it the CRD from AutoML?

How did you create this Experiment? How can I reproduce what you saw?

**Comment by DHolmanCoding** [comment]
> Apologies for being completely unsynced but what exactly do you mean by `Experiment`? Is it the CRD from AutoML?
> 
> How did you create this Experiment? How can I reproduce what you saw?

@kimwnasptd The original poster mentions creating an Experiment using the UI within the Kubeflow Central Dashboard. For example you can see the "Experiments" panel in the image below, and this is where they and other with this issue have navigated to create experiments. 
![image](https://user-images.githubusercontent.com/35512326/135949462-2f471fea-7f34-4f9d-99bf-9bbfb7bbf9bc.png)

**Comment by sonfire86** [comment]
+

**Comment by gcheron** [comment]
I am also facing same issue! Were you able to resolve this?

**Comment by Mike-no** [comment]
Same issue from the GUI
Also, the runs of my pipelines don't appear on the Dashboard but I can see the relative pods running on the k8s cluster (microk8s in my case).

**Comment by pshegde** [comment]
+1

**Comment by azuredevopsguy** [comment]
I have experienced this issue on AKS 1.21.1 and Juju install Kubeflow 1.3

First work around I found was to use the kfp command line
kfp pipeline upload 
kfp run submit
The preinstalled pipelines work successfully, this told me the Kubeflow install is sound and the issue was with something else.

To make the Kubeflow UI work, I tried a lot of things.  I suspected the issue is to do with profiles permissions and the experiment 3 error is really saying this user doesn't have permission to run experiments.

Running this command
juju remove-application kubeflow-profiles  --force

Allows for the UI to work and point and click experiments to proceed

Running this command
juju deploy kubeflow-profiles 
then breaks the UI.  

Perhaps removing Kubeflow-profiles from your Kubeflow install will do as a workaround for you.
